src/test/scala/cromwell/util/SampleWdl.scala

```diff
@@ -890,6 +890,63 @@ object SampleWdl {
     override val rawInputs = Map.empty[String, String]
   }
 
+  object InputIsolationWdl extends SampleWdl {
+    override def wdlSource(runtime: String = "") =
+      """
+        |task isolated {
+        |    File fileInTask
+        |    Array[File] otherFilesInTask
+        |    command {
+        |        ls -1 "$(dirname ${fileInTask})" | wc -l | tr -d '[[:space:]]'
+        |    }
+        |    output {
+        |        String ls = read_string(stdout())
+        |        Array[String] content = read_lines(fileInTask)
+        |    }
+        |    RUNTIME
+        |}
+        |
+        |task produceFile {
+        |    command {
+        |        echo "some text" > out
+        |    }
+        |    output {
+        |        File out = "out"
+        |    }
+        |    RUNTIME
+        |}
+        |
+        |workflow wf {
+        |    Array[File] filesNotPassedToTask
+        |    File filePassedToTask
+        |    Array[File] otherFilesPassedToTask
+        |
+        |    call isolated { input:
+        |                       fileInTask = filePassedToTask,
+        |                       otherFilesInTask = otherFilesPassedToTask
+        |                  }
+        |    call produceFile
+        |    call isolated as butnotalone { input:
+        |                                      fileInTask = produceFile.out,
+        |                                      otherFilesInTask = otherFilesPassedToTask
+        |                                 }
+        |}
+      """.stripMargin.replaceAll("RUNTIME", runtime)
+
+    val tempDir = Files.createTempDirectory("InputFiles")
+    val taskFile = createCannedFile(prefix="task", contents="", dir=Some(tempDir))
+    val otherTaskFile1 = createCannedFile(prefix="othertask1", contents="", dir=Some(tempDir))
+    val otherTaskFile2 = createCannedFile(prefix="othertask2", contents="", dir=Some(tempDir))
+    val workflowOnlyFile1 = createCannedFile(prefix="workflow1", contents="", dir=Some(tempDir))
+    val workflowOnlyFile2 = createCannedFile(prefix="workflow2", contents="", dir=Some(tempDir))
+
+    override val rawInputs = Map(
+      "wf.filePassedToTask" -> taskFile.getAbsolutePath,
+      "wf.otherFilesPassedToTask" -> Seq(otherTaskFile1.getAbsolutePath, otherTaskFile2.getAbsolutePath),
+      "wf.filesNotPassedToTask" -> Seq(workflowOnlyFile1.getAbsolutePath, workflowOnlyFile2.getAbsolutePath)
+    )
```

Comment:
Whoops please ignore what I said completely, we do put all those files in the call dir on the local backend.  :frowning: